docs/usage.md

```diff
@@ -8,24 +8,29 @@ There are two ways to use the generator:
 - [Generator library](#using-as-a-modulepackage)
 
 ## AsyncAPI CLI
+Generates whatever you want using templates compatible with AsyncAPI Generator.
 ```bash
-Usage: asyncapi generate fromTemplate <asyncapi> <template> [<options>]
+USAGE
+  $ asyncapi generate fromTemplate [ASYNCAPI] [TEMPLATE] [-h] [-d <value>] [-i] [--debug] [-n <value>] [-o <value>] [--force-write] [-w] [-p <value>] [--map-base-url <value>]
 
-- <asyncapi>: Local path or URL pointing to AsyncAPI document for example https://bit.ly/asyncapi
-- <template>: Name of the generator template like for example @asyncapi/html-template or https://github.com/asyncapi/html-template
+ARGUMENTS
+  ASYNCAPI  - Local path, url or context-name pointing to AsyncAPI file
+  TEMPLATE  - Name of the generator template like for example @asyncapi/html-template or https://github.com/asyncapi/html-template
 
```

Comment:
@derberg correct me if I am wrong but I feel the changes being made to this file might be unnecessary especially this part `asyncapi generate fromTemplate [ASYNCAPI] [TEMPLATE] [-h] [-d <value>] [-i] [--debug] [-n <value>] [-o <value>] [--force-write] [-w] [-p <value>] [--map-base-url <value>]` why not just leave it as was `asyncapi generate fromTemplate <asyncapi> <template> [<options>]`

Reply:
See earlier argument. Keep documentation in sync with help-output generated from code

Reply:
yeah, I think in this particular case good to follow the output from CLI that is following best practices